brianlong/www.validators.app
# 183002645_update_locations_for_reliablesite

#### What's this PR do?
- update locations for reliablesite

#### How should this be manually tested?
- Run the script locally, it should update your reliablesite validators data centers

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183002645)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
